code-challenges.test.js

```diff
@@ -185,27 +192,27 @@ const vowelTester3 = "challenges"
 // Expected output: 2
 
 
-//  describe("firstVowelIndex",() => {
-//      it("takes in a string and logs the index of the first vowel.", () => {
-//          expect (firstVowelIndex(vowelTester1)).toEqual(1)
-//          expect (firstVowelIndex(vowelTester2)).toEqual(0)
-//          expect (firstVowelIndex(vowelTester3)).toEqual(2)
-//           })
-//          })
+ describe("firstVowelIndex",() => {
+     it("takes in a string and logs the index of the first vowel.", () => {
+         expect (firstVowelIndex(vowelTester1)).toEqual(1)
+         expect (firstVowelIndex(vowelTester2)).toEqual(0)
+         expect (firstVowelIndex(vowelTester3)).toEqual(2)
+          })
+         })
 
 
 
 // // b) Create the function that makes the test pass.
 
 
-// const firstVowelIndex = (string) =>{ 
-//     const vowels = ['a', 'e', 'i', 'o', 'u']
-//     for (let i = 0; i < string.length; i++) {
-//         if(vowels.includes(string[i])) {
-//             return i
-//         }
-//     }
-// }
+const firstVowelIndex = (string) =>{ 
+    const vowels = ['a', 'e', 'i', 'o', 'u']
+    for (let i = 0; i < string.length; i++) {
+        if(vowels.includes(string[i])) {
+            return i
```

Comment:
Your pseudo code does not explain what is happening in the code. You said that you want to iterate through the string and find the first vowel and return the index, but the point of the pseudo code is to explain how that is happening.